exonum-java-binding-core/rust/ejb-app/src/node_builder.rs

```diff
@@ -118,9 +118,6 @@ mod tests {
     fn broken_config() {
         let cfg = create_config("broken.toml", "not_list = 1");
         let services_to_enable = services_to_enable(cfg);
```

Comment:
Unused variable seems to break clippy (shall it be prefixed with an underscore?).